requirements.txt

```diff
@@ -10,3 +10,6 @@ numpy==1.15.4
 pyee==5.0.0
 aiohttp==3.4.4
 # websocket-client==0.54.0
+bitfinex-api-py==1.1.10
+peewee==3.14.2
+nest-asyncio==1.5.1
```

Comment:
> pip3 install -r requirements.txt

```
ERROR: Cannot install bitfinex-api-py==1.1.10 and pyee==5.0.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested pyee==5.0.0
    bitfinex-api-py 1.1.10 depends on pyee~=8.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```

Reply:
version of pyee updated to 8.0.1 in requirements